scenario/elasticreload.go

```diff
@@ -19,27 +18,13 @@ import (
 
 type (
 	// ElasticReloadCore Currently used ElasticReloadCore (as opposed to deprecated settings)
-	ElasticReloadCore struct {
-		// PollInterval time in-between polling for reload status
-		PollInterval helpers.TimeDuration `json:"pollinterval" displayname:"Poll interval" doc-key:"elasticreload.pollinterval"`
-		// PollingOff turns polling for status off except after event websocket reconnect
-		PollingOff bool `json:"pollingoff" displayname:"Polling off" doc-key:"elasticreload.pollingoff"`
-	}
+	ElasticReloadCore struct{}
```

Comment:
Is there a reason to keep this empty struct around?

Reply:
hmm not directly. Might be good to live it in case we want to add other parameters in the future since we already have the duouble core things working here (considering the unmarshal bug if you don't do it this way). Other than that it could actually be removed.

Reply:
If it solves a problem it's fine. We shouldn't keep it around just because we might want it in the future - we already have it in the git history.

Reply:
not because we "might need it" more because it's easy to think that adding a field to the main struct will work, but it won't because of the bug in json.Unmarshal, so will remind of that saving hours of debugging if we forget. But sure, wasn't directly intentional to leave it, just figured now that maybe it was a good idea, but if no-one agrees I'll remove it.

Reply:
Ah right. Now I see what you mean.